logbook-logstash-logback-encoder/src/main/java/org/zalando/logbook/lle/LogbackLogstashLogWriter.java

```diff
@@ -0,0 +1,22 @@
+package org.zalando.logbook.lle;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+import java.io.IOException;
+
+import org.apiguardian.api.API;
+import org.slf4j.Marker;
+import org.zalando.logbook.Correlation;
+import org.zalando.logbook.Precorrelation;
+
+@API(status = EXPERIMENTAL)
+public interface LogbackLogstashLogWriter {
```

Comment:
Do you expect other implementations? I'd actually get rid of this an inline the default implementation into your sink.

Reply:
Or alternatively apply the marker within your custom writer but without the need to change the interface, i.e. stick with `HttpLogFormatter`.

Reply:
The alternative will probably not work since you need access to the request/response to construct your message. But your use case i pretty much what Sink was designed for.

Reply:
Remved interface